lib/routes/dtfb.js

```diff
@@ -1,35 +1,79 @@
 const router = require('express').Router();
 const fetch = require('node-fetch');
 
+async function getTournamentDates() {
+  const result = await fetch('https://dtfb.de/index.php/termine');
+  const html = await result.text();
+  var reg = new RegExp(/(3[01]|[12][0-9]|0?[1-9])\.(1[012]|0?[1-9])\.\s/g);
+  const tournamentDates = html.match(reg);
+  return tournamentDates;
+}
+
+async function getTournamentNames() {
+  const result = await fetch('https://dtfb.de/index.php/termine');
+  const html = await result.text();
+  var reg = new RegExp(/(?<=wrap>\s)([^<]+)(?=\s<[br|/td])/g);
+  const tournamentNames = html.match(reg);
+  const cleanedTournamentNames = tournamentNames
+    .filter((tournamentNames) => tournamentNames)
+    .map((tournamentNames) => {
+      return tournamentNames.trim();
+    });
+  // eslint-disable-next-line no-unused-vars
+  const removeEntries = cleanedTournamentNames.splice(0, 4);
+  return cleanedTournamentNames;
+}
+
+async function getTournamentLocations() {
+  const result = await fetch('https://dtfb.de/index.php/termine');
+  const html = await result.text();
+  var reg = new RegExp(/(?<=\s<td nowrap>)(.*)(?=<\/td>[\s]{2,}<td nowrap>)/g);
+  const tournamentLocations = html.match(reg);
+  const cleanedTournamentLocations = tournamentLocations.map(
+    (tournamentLocations) => {
+      return (
+        tournamentLocations
+          // enable äöüß
```

Comment:
you have this comment inside a return function... make sure it doesn't cause any troubles since it's not greyed out. do you really need it?